substrate/client/transaction-pool/src/fork_aware_txpool/fork_aware_txpool.rs

```diff
@@ -1196,7 +1291,7 @@ where
 							resubmitted_to_report += 1;
 
 							if !contains {
-								log::trace!(
+								tracing::trace!(
 									target: LOG_TARGET,
 									"[{:?}]: Resubmitting from retracted block {:?}",
```

Comment:
```suggestion
                                    tx_hash,
									"Resubmitting from retracted block",
```

And `hash` should be `block_hash` or `at`.